### ISSUE TITLE
Consider adding a `preserve` option

### ISSUE THREAD

**Issue by pascalduez** [pascalduez/postcss-apply#14]
https://github.com/postcss/postcss-custom-properties#preserve

**Comment by kevinSuttle** [comment]
I've noticed that when you combine this plugin with postcss-custom-properties and add `preserve: true`, you get double CSS Custom Properties, but nothing preserved, which triggers the error found here: https://github.com/pascalduez/postcss-apply/blob/master/src/visitor.js#L57

### No preserve option set in postcss-custom-properties


```js
new webpack.LoaderOptionsPlugin({
        options: {
          postcss: function (webpack) {
            return [
              cssApply,
              customProperties({
                variables: colorPalettes, // JS object from file
                warnings: false
              }),
              autoprefixer,
            ]
          }
        }
```


<img width="833" alt="screen shot 2016-09-30 at 2 17 26 am" src="https://cloud.githubusercontent.com/assets/95672/18982490/94e988bc-86b4-11e6-9afe-7e6de20ef2c3.png">

### `preserve: true` option set in postcss-custom-properties

```diff
new webpack.LoaderOptionsPlugin({
        options: {
          postcss: function (webpack) {
            return [
              cssApply,
              customProperties({
                variables: colorPalettes, 
+               preserve: true,
                warnings: false
              }),
              autoprefixer,
            ]
          }
        }
```

<img width="835" alt="screen shot 2016-09-30 at 2 16 50 am" src="https://cloud.githubusercontent.com/assets/95672/18982521/d8af8664-86b4-11e6-827c-ef566e09ea59.png">

**Comment by pascalduez** [comment]
I'll try to add more unit tests with `postcss-custom-properties` options.

When reporting, please provides simple cases/samples/tests reproducing the issue. Without any Webpack, Storybook and such. This is not a proper way to report bugs.

**Comment by pascalduez** [comment]
Available from version `0.5.0` on.
